src/uct/api/v2/uct_v2.h

```diff
@@ -131,6 +131,17 @@ typedef enum {
     UCT_MD_MEM_DEREG_FIELD_COMPLETION = UCS_BIT(2)  /**< comp field */
 } uct_md_mem_dereg_field_mask_t;
 
+/**
```

Comment:
one more space line before

Reply:
fixed with force pushed commit

Reply:
pls do not fix by force push during review
![image](https://user-images.githubusercontent.com/2255631/126071895-e7b8b7f2-99b1-4f1d-bfe7-35afc5b58e4e.png)